CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(sycl-gtx-solution)
 # Requirements
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Why is this needed?

Reply:
This option forces ISO C++11 standard, without additional extensions (such as those in GNU's C++11 standard).